src/libs/actions/CloseAccount.js

```diff
@@ -6,7 +6,7 @@ import CONST from '../../CONST';
  * Clear CloseAccount error message to hide modal
  */
 function clearError() {
-    Onyx.merge(ONYXKEYS.CLOSE_ACCOUNT, {error: ''});
+    Onyx.merge(ONYXKEYS.CLOSE_ACCOUNT, {error: '', errors: {}});
```

Comment:
We usually reset the key to `null` instead of `{}`, because the default props in components can set it back to an empty object, and setting it to `null` allows Onyx to delete that key. I think that happens [here](https://github.com/Expensify/react-native-onyx/blob/4efd75ce4cdc0b2e9c1ac0542565e90164572f7b/lib/Onyx.js#L806-L809).   Here's an example in App where we do this. https://github.com/Expensify/App/blob/4dc7fef0ec778be71114898bdf7d56ed363bee0a/src/libs/actions/ReimbursementAccount/errors.js#L30